### PR TITLE
add ConnectEmbed + Fix double connection issue with Safe/Smart in ConnectWallet

### PR DESCRIPTION
**Pull request by MananTank**
* Add ConnectEmbed component to embed the ConnectWallet Modal's content in the page
* Fix the double connection issue when connecting Safe or Smart wallet when using the ConnectWallet / ConnectEmbed component by not setting the personal wallet as the connected wallet and saving the connection information of the personal wallet in a separate state

### CODE REVIEW

File: packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx [hunk 1]
```diff
@@ -227,6 +228,16 @@ export function ThirdwebWalletProvider(
...
 
+  /**
+   * Clear the states of hidden connection
+   */
+  const disconnectHiddenConnection = useCallback(() => {
+    setSigner(undefined);
+    setActiveWallet(undefined);
+    setActiveWalletConfig(undefined);
+    setConnectionStatus("disconnected");
+  }, []);
```
Comment (iketw) [inline packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx:239]: what is a hidden connection?

Comment (MananTank) [reply packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx:239]: @iketw - ignore this. I'm gonna completely change this

File: packages/react/src/evm/locales/es.ts [hunk 2]
```diff
@@ -54,6 +54,7 @@ export function esDefault(): ThirdwebLocale {
       personalWallet: "Cartera personal",
       smartWallet: "Cartera inteligente",
       or: "O",
+      goBackButton: "Back", // TODO
```
Comment (joaquim-verges) [inline packages/react/src/evm/locales/es.ts:57]: Atras

File: packages/react/src/evm/locales/ja.ts [hunk 3]
```diff
@@ -53,6 +53,7 @@ export function jaDefault(): ThirdwebLocale {
       personalWallet: "パーソナルウォレット",
       smartWallet: "スマートウォレット",
       or: "または",
+      goBackButton: "Back", // TODO
```
Comment (joaquim-verges) [inline packages/react/src/evm/locales/ja.ts:56]: gtranslate this?

File: packages/react/src/evm/providers/wallet-ui-states-provider.tsx [hunk 4]
```diff
@@ -20,6 +20,7 @@ export type ModalConfig = {
     onLogin?: (token: string) => void;
     onLogout?: () => void;
   };
+  isEmbed?: boolean;
```
Comment (joaquim-verges) [inline packages/react/src/evm/providers/wallet-ui-states-provider.tsx:23]: so the API is

`<ConnectWallet isEmbed=true />` ?

Comment (MananTank) [reply packages/react/src/evm/providers/wallet-ui-states-provider.tsx:23]: No - It's just `<ConnectEmbed />`

Comment (MananTank) [reply packages/react/src/evm/providers/wallet-ui-states-provider.tsx:23]: This is internal code

**Pull request merged**